swatch-metrics/src/main/resources/application.yaml

```diff
@@ -23,6 +31,8 @@ SERVICE_INSTANCE_INGRESS_TOPIC: platform.rhsm-subscriptions.service-instance-ing
   SPLUNKMETA_namespace: local
   SPLUNK_HEC_INCLUDE_EX: true
   quarkus:
+    management:
+      enabled: false
```

Comment:
Why disable the management server in dev mode? I prefer starting up this server when running in DEV mode.

Reply:
It's  not disabled in the way you are thinking.  That option is very confusingly named.  Setting `quarkus.management.enabled` just moves a bunch of management endpoints from `:8000/q/whatever` to `:9000/q/whatever`.  So setting `quarkus.management.enabled: true` here actually preserves the behavior you like.